builds/e2e/longhaul.yaml

```diff
@@ -90,6 +90,7 @@ jobs:
           test.useTRC: '$(useTRC)'
           test.deploymentFileName: 'long_haul_deployment.template.json'
           testResultCoordinator.storageAccountConnectionString: '$(EdgeLonghaulStorageAccountConnString)'
+          clientModuleTransportType: 'All'
```

Comment:
Do we need for the nested edge as well?

Reply:
I added it to the nested longhaul deploy script.